docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
All I'm saying is that the "traditional" use of a `.env` file is to store secrets that are specific to the particular installation, and that in this usage model the `.env` file should obviously not be committed to source control. This example goes against this traditional usage, by creating an incentive for users to commit the file.

Flask config files are different in my opinion. Those can have non-sensitive values. Sensitive values in config files should import from the environment (such as the `.env` file that should not be on source control).